generators/vue/files-vue.mjs

```diff
@@ -88,6 +87,7 @@ export const vueFiles = {
         'router/index.ts',
         'router/admin.ts',
         'router/pages.ts',
+        'test-setup.ts',
```

Comment:
Probably best fit at `src/main/webapp` instead of `src/main/webapp/app`.

Reply:
I'm not sure its appropriate, currently webapp folder does not have anything vue related and this test app is only required for vue

Reply:
What about src/main/webapp/test?
The microfrontend test files can be moved there too.

Reply:
In react test setup file is in the `src/main/webapp/app` folder and the microfrontend stuff is in `src/main/webapp/microfrontend` folder and for angular I dont see any test setup or microfrontend test files so IMO it would be consistent to leave the test-setup file here else we should try and make these consistent across frameworks

Reply:
Indeed setup is in `src/main/webapp/app` and microfrontend test files in `/src/main/webapp/microfrontend`.

Should microfrontend test files be moved to `src/main/webapp/microfontend`?

Reply:
ya that makes sense

Reply:
i moved them to `src/main/webapp/microfontend`